examples/artemis/artemis_single.yaml

```diff
@@ -2,7 +2,3 @@ apiVersion: broker.amq.io/v1beta1
 kind: ActiveMQArtemis
 metadata:
   name: artemis-broker
-spec:
-  deploymentPlan:
-    size: 1
```

Comment:
Should the size field be preserved  at least in 1 example?

Reply:
added a new example with `size: 0`